restaurants.py

```diff
@@ -279,6 +283,96 @@ def parse_menu(self, day):
         return menu
 
 
+class TOTORestaurant(Restaurant):
+    def __init__(self, session) -> None:
+        super().__init__()
+        self.aio_session = session
+        self.content = None
+        self.name = 'TOTO (4.9€ / 4.2€ bez polievky / 6.2€ extra menu / 7.4€ business menu)'
+        self.url = 'http://www.totorestaurant.sk/'
+
+    async def retrieve_menu(self, day=TODAY) -> Menu:
+        async with self.aio_session.get(self.url) as resp:
+            self.content = BeautifulSoup(await resp.text(), 'html.parser')
+            return self.parse_menu(day)
+
+    def parse_menu(self, day):
+        menu = Menu(self.name)
+        date_div = self.content.select('div.date')[day]
+
+        for sibling in date_div.next_siblings:
+            if isinstance(sibling, element.NavigableString):
+                continue
+            if sibling.name in ['h2', 'div']:
+                break
+            text = sibling.text.strip()
+            if text:
+                menu.add_item(text)
+
+        return menu
+
+
+class AvalonRestaurant(Restaurant):
+    def __init__(self, session) -> None:
+        super().__init__()
+        self.aio_session = session
+        self.content = None
+        self.name = 'Avalon'
+        self.url = 'https://avalonrestaurant.sk/denne-menu/'
+
+    async def retrieve_menu(self, day=TODAY) -> Menu:
+        async with self.aio_session.get(self.url) as resp:
+            self.content = BeautifulSoup(await resp.text(), 'html.parser')
+            return self.parse_menu(day)
+
+    def parse_menu(self, day):
+        menu = Menu(self.name)
+        today_menu = self.content.select('section.article__content')[day]
+        for p in today_menu.find_all('p'):
+            text = p.text.strip()
+            if text:
+                menu.add_item(text)
+
+        return menu
+
+
+class CasaInkaRestaurant(Restaurant):
+    def __init__(self, session) -> None:
+        super().__init__()
+        self.aio_session = session
+        self.content = None
+        self.name = 'Casa Inka (5.5€ / 6€ špecialita)'
+        self.url = 'http://www.casa-inka.sk/index.php?page=jedalny&kategoria=menu'
+
+    async def retrieve_menu(self, day=TODAY) -> Menu:
+        async with self.aio_session.get(self.url) as resp:
+            self.content = BeautifulSoup(await resp.text(), 'html.parser')
+            return self.parse_menu(day)
+
+    def parse_menu(self, day):
+        raise NotImplementedError
```

Comment:
It would be nicer if we didn't retrieve the menu at all when we don't intend to parse it.